### PR TITLE
Docs: update docs for `src validate` command to add new `install` subcommand

### PR DESCRIPTION
**Pull request by jdpleiness**
localhost_5080_admin_validation

This is a documentation update for the new `install` subcommand for `src validate` that will be introduced in https://github.com/sourcegraph/src-cli/pull/921.

- [x] Updated examples to reflect new configuration layout
- [x] Updated references to `src validate` to be `src validate install`
- [x] Made warning that features are experimental more visible

## Test plan

Run `yarn docsite:serve` to stand up documentation.

### CODE REVIEW

File: doc/admin/validation.md [hunk 1]
```diff
@@ -1,21 +1,22 @@
 # Sourcegraph Instance Validation
 
->NOTE: **Sourcegraph Instance Validation is currently experimental.** We're exploring this feature set. 
+>🚨 WARNING 🚨: **Sourcegraph Instance Validation is currently experimental.** We're exploring this feature set.
```
Comment (abeatrix) [inline doc/admin/validation.md:3]: We should use emoji more 😆

**Review by abeatrix — APPROVED** [review]
LGTM 🔥

**Comment by abeatrix** [comment]
What do you think about adding those nice code snippet to the docs as examples 😏? I also like how clean it currently is though 🥴

**Comment by jdpleiness** [comment]
I am going back and forth on that too. I think for now leave it as is, then we can always add more later.

**Pull request merged**